### PR TITLE
VDB-1437 Changes naming for vow file auction attributes

### PR DESCRIPTION
**Pull request by yaoandrew**
NOTE: This contains breaking changes as we are renaming tables in an old migration.

### CODE REVIEW

**Comment by rmulhol** [comment]
@yaoandrew was about to review this but holding off bc nobody's tagged - lmk if you're ready for review or need help working through anything

File: transformers/test_data/vow_file.go [hunk 1]
```diff
@@ -48,13 +48,13 @@ var rawVowFileLog = types.Log{
 var VowFileEventLog = core.EventLog{
```
Comment (rmulhol) [inline transformers/test_data/vow_file.go:48]: Super minor but maybe worth renaming this to specify `AuctionAttributes` as well

File: transformers/test_data/vow_file.go [hunk 2]
```diff
@@ -48,13 +48,13 @@ var rawVowFileLog = types.Log{
...
 	ID:          int64(rand.Int31()),
 	HeaderID:    int64(rand.Int31()),
-	Log:         rawVowFileLog,
+	Log:         rawVowFileAuctionAttributesLog,
 	Transformed: false,
 }
 
 var VowFileModel = event.InsertionModel{
```
Comment (rmulhol) [inline transformers/test_data/vow_file.go:55]: Same here - also wondering if we might want to use `CopyModel` to hedge against test pollution

**Review by rmulhol — APPROVED** [review]
Nothing blocking - could be cool to get the story number in the PR title so it shows up in the changelog

:shipit:

File: transformers/test_data/vow_file.go (hunk 2 above)
Comment (yaoandrew) [reply transformers/test_data/vow_file.go:55]: Cool. I noticed this and got these in #251

**Pull request merged**